search/docker-compose.yml

```diff
@@ -217,6 +221,106 @@ services:
       driver: "json-file"
     restart: always
 
+  cvat_db:
```

Comment:
(We should really take the opportunity to factor this out into a library of services shared between dev and prod.))